tls/s2n_connection_serialize.c

```diff
@@ -15,9 +15,19 @@
 
 #include "tls/s2n_connection_serialize.h"
 
+#include "crypto/s2n_sequence.h"
 #include "tls/s2n_connection.h"
 #include "tls/s2n_tls13_key_schedule.h"
 
+static bool s2n_libcrypto_supports_evp_aead_tls(void)
```

Comment:
Yeah probably good to keep those consistent

Reply:
Oh yeah I waffled on that _a lot_. It's probably best for a follow up PR because wasn't 100% sure those two macros signify the same thing? And I didn't want to have that argument in this PR. Plus its a larger change. I'll open an issue.

Reply:
I mean minimally S2N_AEAD_AES_GCM_AVAILABLE needs to consider your new macros, because we actually call the methods you're checking for based on S2N_AEAD_AES_GCM_AVAILABLE.